packages/owl-bot/src/handlers.ts

```diff
@@ -167,34 +167,41 @@ export async function scanGithubForConfigs(
   configsStore: ConfigsStore,
   octokit: OctokitType,
   githubOrg: string,
-  orgInstallationId: number,
-  logger = console
+  orgInstallationId: number
 ): Promise<void> {
-  // Some configurations may not have an installationId yet.
-  // Revisit them after we have collected an installationId.
-  type refreshFunction = (installationId: number) => Promise<void>;
-  const refreshLaters: refreshFunction[] = [];
-
-  const repos = octokit.repos.listForOrg.endpoint.merge({org: githubOrg});
```

Comment:
I had a hiccup the first time I tried to iterate through all repos using this approach, things worked as expected using the approach documented in the restify docs (_the code I migrated to_).